koji_fedoramessaging_messages/package.py

```diff
@@ -60,36 +60,40 @@ class ListChangeV1(KojiFedoraMessagingMessage):
 
     @property
     def force(self) -> Optional[bool]:
-        return self.body["force"]
+        return self.body.get("force")
```

Comment:
In the JSON schema, the fields are not marked in a `required` list. We have two options:
- either mark them as required, and we can access them by key in the properties
- handle their absence in the properties with a `.get()` call

I have no preference as I don't know whether Koji always gives this data.

The same applies to the rest of the file and a couple others.